src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private File file;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeDataToFile(readAndSort(fromFileName), toFileName);
+
+    }
+
+    public int [] readAndSort(String fromFileName) {
+        file = new File(fromFileName);
+        String [] temp = new String[2];
+        int [] result = new int[3];
```

Comment:
The `result` array should have informative names for each index to make the code more readable. Consider using constants or an enum to represent the indices, or switch to a class or structure that has named fields.